resources/views/components/ticket/ticket-list.blade.php

```diff
@@ -76,7 +76,7 @@
                             <td>{!! userAvatar($ticket->author ?? 'Deleted User') !!}</td>
                             <td>{!! userAvatar($ticket->reporter ?? 'Deleted User') !!}</td>
                             <td class="smalldate">{{ getNiceDate($ticket->ReportedAt->unix()) }}</td>
-                            @if ($showResolver)
+                            @if ($showResolver && !TicketState::isOpen($ticket->ReportState))
```

Comment:
Need an empty `<td>` if the column exists:
```
@if ($showResolver)
    @if (TicketState::isOpen($ticket->ReportState))
        <td />
    @else
        <td>{!! userAvatar($ticket->resolver ?? 'Deleted User') !!}</td>
    @endif
@endif
```

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c91a8452-4abc-4b23-b586-96d06d1669e8)

Reply:
Yep. Fixed in latest.